ceph.spec.in

```diff
@@ -27,7 +27,7 @@
 # no gperftools/tcmalloc on s390(x)
 %bcond_with tcmalloc
 %endif
-%bcond_with lowmem_builder
+%bcond_without lowmem_builder
```

Comment:
To answer your question: no, the memory resource usage is controlled by the parallelism code in the spec file, which sets the `-j` value to match the amount of memory available in the build worker. This is not controlled by any bcond.